test/integration/graphql/partners_integration_test.rb

```diff
@@ -253,4 +253,29 @@ def check_basic_fields(data, partner)
     partner_data = data['partnersByTag']
     assert partner_data.length == 1, 'expecting to see a tag on this partner'
   end
+
+  test 'returns null properly if openning times are missing' do
+
+    partner = FactoryBot.create(:partner, opening_times: nil)
+
+    query_string = <<-GRAPHQL
+      query {
+        partner(id: #{partner.id}) {
+          openingHours {
+            opens
+            closes
+            dayOfWeek
+          }
+        }
+      }
+    GRAPHQL
+
+    result = PlaceCalSchema.execute(query_string)
+    assert result.key?('errors') == false, 'errors are present'
```

Comment:
I think you can use `refute` here rather than `assert x == false`?

Reply:
do you think i should change it?

Reply:
It's up to you, and how much energy you have IMO. Like I think changing it would mean changing the whole file, but on the other hand `assert x == false` is generally an antipattern I think